app/workers/users/estimate_default_language_worker.rb

```diff
@@ -1,6 +1,8 @@
 module Users
-  class EstimateDefaultLanguageJob < ApplicationJob
-    queue_as :users_estimate_language
+  class EstimateDefaultLanguageWorker
```

Comment:
Changing this job to a worker is great but you also need to update the call we make to use the new worker in the user model.
```ruby
def estimate_default_language
  Users::EstimateDefaultLanguageWorker.perform_async(id)
end
```

Further, you will notice that this method is called in an `after_create` callback. At that point, the user is not committed to the database so the worker will not be able to find it. For this reason, we need to change the callback to `after_create_commit`

Reply:
got it, I will do it